Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+
+Finally, we want to translate the origin of the tracker so that it's position appears to be at the center of the eyes. It's mounted a few inches above and in-front of the eyes, so we'll need to translate it towards the head and down in the server config file:
+````json
+"aliases": {
+	"/me/head": {
+      "posttranslate": [0.0, -0.1143, -0.0635],
+      "rotate": {
+        "axis": "x",
+        "degrees": -90
+      },
+      "child": {
+        "changeBasis": {
+          "x": "x",
+          "y": "y",
+          "z": "z"
+        },
+        "child": "/me/puck"
+      }
+    }
+}
+````
+Note: The posttranslate values here may need some fine-tuning depending on exact location of the tracking sensor in relation to the center of the eyes. The units are meters.
+
+Tip: In addition to rotate and posttranslate, there's also postrotate and translate. Experiment to see how this changes tracking.
+
+All together, our config file with Vive Puck tracking and HDK2 display looks like:
+````json
+{
+	"display": "displays/OSVR_HDK_2_0.json",
+	"renderManagerConfig": {
+		"meta": {
+			"schemaVersion": 1
+		},
+		"renderManagerConfig": {
+			"directModeEnabled": true,
+			"directDisplayIndex": 0,
+			"directHighPriorityEnabled": true,
+			"numBuffers": 2,
+			"verticalSyncEnabled": true,
+			"verticalSyncBlockRenderingEnabled": true,
+			"renderOverfillFactor": 1.2,
+			"renderOversampleFactor": 1.0,
+
+			"window": {
+				"title": "OSVR",
+				"fullScreenEnabled": true,
+				"xPosition": 1920,
+				"yPosition": 0
+			},
+
+			"display": {
+				"rotation": 180,
+				"bitsPerColor": 8
+			},
+
+			"prediction": {
+				"enabled": true,
+				"staticDelayMS": 26,
+				"leftEyeDelayMS": 0,
+				"rightEyeDelayMS": 0,
+				"localTimeOverride": true
+			},
+
+			"timeWarp": {
+				"enabled": true,
+				"asynchronous": false,
+				"maxMsBeforeVSync": 5
+			}
+		}
+	},
+
+	"aliases": {
+		"/me/head": {
+			"posttranslate": [0.0, -0.1143, -0.0635],
+			"rotate": {
+				"axis": "x",
+				"degrees": -90
+			},
+			"child": {
+				"changeBasis": {
+					"x": "x",
+					"y": "y",
+					"z": "z"
+				},
+				"child": "/me/puck"
+			}
+		}
+	}
+}
+````
+It's really easy to switch the HDK2 display for any other OSVR HMD -- just swap the name of the display:
+````json
+"display": "displays/OSVR_HDK_2_0.json",
+````
+could be changed to work with the Sensics dSight display:
+````json
+"display": "displays/Sensics_dSight_landscape_1input_sbs.json",
+````
+Reminder that if you're using OSVRTrackerView to visualize tracked devices, only /me/head, /me/hands/left, and /me/hands/right are visualized by default. To see other devices, specify their path in the command line. For example, to launch OSVRTrackerView and see four connected OSVR-Vive devices (Vive HMD, Controllers, Puck), launch OSVRTrackerView with the following command:
+````c
+OSVRTrackerView.exe --pose /me/head /me/hands/left /me/hands/right /me/puck
```

Comment:
I think you need to specify `--pose` for each path individually, don't you? `OSVRTrackerView.exe --pose /me/head --pose /me/hands/left ...`